pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/AdditionTransformFunction.java

```diff
@@ -45,6 +45,7 @@ public String getName() {
 
   @Override
   public void init(List<TransformFunction> arguments, Map<String, ColumnContext> columnContextMap) {
+    super.initArguments(arguments);
```

Comment:
(minor) If we use `super.`, suggest making the base override `init()` as well with the same arguments